server/src/main/java/com/objectcomputing/checkins/CheckIns.java

```diff
@@ -0,0 +1,83 @@
+package com.objectcomputing.checkins;
+
+import java.sql.Date;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="record_checkins")
+public class CheckIns {
+
+    public CheckIns() {}
+
+    public CheckIns(String teamMember, UUID pdlId, Date checkInDate, int targetQtr, int targetYear) {
+        this.teamMember= teamMember;
+        this.pdlId=pdlId;
+        this.checkInDate=checkInDate;
+        this.targetQtr=targetQtr;
+        this.targetYear=targetYear;
+    }
+    
+    @Column(name="teamMember")
+    private String teamMember;
```

Comment:
@mkimberlin remind me what this was for ? I couldn't remember so just added as a String.

Reply:
This is the reference to the team member ID. Similar to PDL ID below.  Team Member ID + PDL ID + Date = A Check-in